cloudsql/mysql/database-sql/README.md

```diff
@@ -0,0 +1,107 @@
+
+### Windows/PowerShell
+The Cloud SQL Proxy currently requires the use of TCP on Windows. The optional ```DB_TCP_HOST``` environment variable can be used to specify the proxy's local address, and the ```INSTANCE_CONNECTION_NAME``` environment variable is not needed in this case.
+```powershell
+$env:GOOGLE_APPLICATION_CREDENTIALS="<CREDENTIALS_JSON_FILE>"
+$env:DB_TCP_HOST="127.0.0.1:3306"
+$env:DB_USER="<DB_USER_NAME>"
+$env:DB_PASS="<DB_PASSWORD>"
+$env:DB_NAME="<DB_NAME>"
+```
+
+## Running locally
+
+To run this application locally, download and install the `cloud_sql_proxy` by
+following the instructions
+[here](https://cloud.google.com/sql/docs/mysql/sql-proxy#install). Once the
+proxy has been downloaded, use the following commands start it running in the background for local testing on either Linux/Mac OS or Windows.
+
+### Launch the proxy on Linux/Mac OS
+
+Linux environments use a Unix socket for the Cloud SQL proxy, so you'll need to create the `/cloudsql`
+directory and give the user running the proxy the appropriate permissions:
+```bash
+sudo mkdir /cloudsql
+sudo chown -R $USER /cloudsql
+```
+
+Once the `/cloudsql` directory is ready, use the following command to start the proxy in the
+background:
+```bash
+./cloud_sql_proxy -dir=/cloudsql --instances=$CLOUD_SQL_CONNECTION_NAME --credential_file=$GOOGLE_APPLICATION_CREDENTIALS
+```
+Note: Make sure to run the command under a user with write access in the 
+`/cloudsql` directory. This proxy will use this folder to create a unix socket
+the application will use to connect to Cloud SQL. 
+
+### launch the proxy on Windows
+
+Windows environments use a TCP connection for the Cloud SQL proxy. Use this PowerShell command to launch the proxy:
+
+```powershell
+Start-Process -filepath "C:\<path to proxy exe>" -ArgumentList "-instances=<project-id>:<region>:<instance-name>=tcp:3306"
+```
+
+Note the the port number is set to ```3306``` in the above example, so the proxy will listen on that port locally. The same port number was used in the ```DB_TCP_HOST``` environment variable we set earlier, which tells the sample code the address and port number for local communication with the proxy. If you want to use a different port number (for example, because a different MySQL instance is using port 3306 locally), you'll need to change both the ```tcp``` parameter in the proxy invocation command and the ```DB_TCP_HOST``` environment variable.
+
+### Testing the application
+
+To test the application locally, follow these steps after the proxy is running:
+
+* Install the MySQL driver: `go get github.com/go-sql-driver/mysql`
+* Run the application: `go run cloudsql.go`
+* Navigate to `http://127.0.0.1:8080` in a web browser to verify your application is running correctly.
+
+## Google App Engine Standard
+
+To run on GAE-Standard, create an App Engine project by following the setup for these 
+[instructions](https://cloud.google.com/appengine/docs/standard/python3/quickstart#before-you-begin).
```

Comment:
python3 -> go link
```suggestion
[instructions](https://cloud.google.com/appengine/docs/standard/go/quickstart#before-you-begin).
```

Reply:
Good catch, fixed in next commit coming shortly.